osu.Game/Screens/Select/Carousel/DrawableCarouselItem.cs

```diff
@@ -25,6 +25,10 @@ public abstract class DrawableCarouselItem : Container
 
         public override bool IsPresent => base.IsPresent || Item.Visible;
 
+        public bool DrawableIsPresent => base.IsPresent;
```

Comment:
Not a fan of this, it'll be intensely confusing to anyone that stumbles upon it. I'd just rather remove and revert the change that uses this.

Reply:
Yeah, this can be reverted, but then the ``alpha == 0`` should be replaced with ``alpha <= 0.0001f`` as that is what ``IsPresent`` checks.